src/Security/BeamNG.cpp

```diff
@@ -301,9 +316,27 @@ void LegitimacyCheck(){
     Result.clear();
     RegCloseKey(hKey);
     //if(TraceBack < 3)exit(-1);
+    #elif defined(__linux__)
+    struct passwd *pw = getpwuid(getuid());
+    std::string homeDir = pw->pw_dir;
+    // Right now only steam is supported
+    std::ifstream libraryFolders(homeDir + "/.steam/root/steamapps/libraryfolders.vdf");
+    auto root = tyti::vdf::read(libraryFolders);
+
+    for (auto folderInfo: root.childs){
+        if (std::filesystem::exists(folderInfo.second->attribs["path"] + "/steamapps/common/BeamNG.drive/")){
+            GameDir = folderInfo.second->attribs["path"] + "/steamapps/common/BeamNG.drive/";
+            break;
+        }
+    }

```

Comment:
This only works for steam as it uses the steam libraryfolders.vdf to find the game. There is no registry on Linux so it would be difficult to find the game otherwise.